tidb-lightning.toml

```diff
@@ -247,8 +248,8 @@ level-1-compact = false
 # if set true, compact will do full compaction to tikv data.
 # if this setting is missing, the default value is false.
 compact = false
-# if set to true, lightning will analyze all table together at last
-analyze-at-last = false
+# if set to true, lightning will run checksum and analyze for all tables together at last
+checksum-at-last = false
```

Comment:
renaming this option is a breaking change. has #509 been included in any release?

Reply:
No, the newest v4.0.9/v3.0.20 doesn't contain this commit

Reply:
in that case maybe choosing a word which represents both "checksum" and "analyze" is better (e.g. `post-process-at-last`). or maybe declare "analyze" means "checksum" as well 🙃

Reply:
I agree with `post-process-at-last`. I think it is not a configuration that could be changed by customer. It could only be adjust by DBA or develop.